src/main/resources/templates/projects/discuss.ftlh

```diff
@@ -44,11 +45,12 @@ Discussion page within Project overview.
                 </#if>
 
                 <div class="push-down">
-                    @utils.editor(
-                    saveCall = routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug),
-                    cancellable = false,
-                    enabled = true
-                    )
+                    <@editor.editor saveCall=routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug) cancellable=false enabled=true/>
+<#--                    @utils.editor(-->
+<#--                    saveCall = routes.getRouteUrl("projects.postDiscussionReply", p.project.ownerName, p.project.slug),-->
+<#--                    cancellable = false,-->
+<#--                    enabled = true-->
+<#--                    )-->
```

Comment:
Please don't just comment out non-ported blocks (with single line comments), that just makes porting them later *ever so slightly slightly* more annoying